library/src/main/java/com/google/android/exoplayer/metadata/id3/APICFrame.java

```diff
@@ -0,0 +1,30 @@
+package com.google.android.exoplayer.metadata.id3;
```

Comment:
These files need to carry the same copyright header comment as the rest of the code base.